sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -20,7 +22,7 @@ import {
   AssessmentHistory,
   StudentAssessmentStatus,
 } from "./student-assessment.models";
-import { WorkflowClientService } from "@sims/services";
+import { Queues, StartAssessmentQueueInDTO } from "@sims/queue";
```

Comment:
Should it be in a completely segregated library or just be placed alongside the shared services?
@ann-aot @guru-aot @andrepestana-aot, thoughts?

Reply:
In mm, IMO, since these are queue specific and it's not shared with another app. can't we have it with the queue app?.let me know if I am missing anything.

Reply:
I was talking about what we need to share, for instance, the `modules` and `DTOs`.

Reply:
I was under the impression that we will be sharing the common interfaces, dto , enum, etc... in the `lib/`

Reply:
@andrewsignori-aot  Do we have plans to use the queues only from the api?

Reply:
@andrewsignori-aot I think we can have it segregated. What was the advantage of placing them along the shared services, other than the service consuming it.

Reply:
@andrepestana-aot queues can be used by any application: API, Workers, Queue-consumers. 

@guru-aot having that alongside the services does not mean that it will be shared only with services. The queues part that is shared is pretty small and it is not supposed to grow, being mostly about connectivity and DTOs.

@ann-aot I am not sure if I was able to follow your comment. Common interfaces and DTOs can be shared from the `@sims/utilities` for instance but I do not see an issue to Queue DTOs be shared from `@sims/services/queue`.